apps/client/.eslintrc.js

```diff
@@ -30,5 +30,8 @@ module.exports = {
     'no-unused-vars': 'off',
     '@typescript-eslint/no-unused-vars': 'error',
     'import/order': ['error', { 'newlines-between': 'always' }],
+    'arrow-body-style': 'off',
+    'react/function-component-definition': 'off',
+    'react/jsx-props-no-spreading': 'off',
   },
```

Comment:
to fix default props eslint error you need to add this rule:
```js
'react/require-default-props': [
      'error',
      {
        functions: 'defaultArguments',
      },
    ],
```

before as a default it requires `defaultProps` param in the function component (it's possible to add this param to the function and React will know about the default values for props) but `functions: 'defaultArguments',` changes it to look for arguments default values 👌 

It's interesting thing about JS - as function in JS is an object it also has params, and like an object, it can have assigned params manually. (`Slider` in the example is a function component):
`Slider.defaultProps = { min: 0 }`

Reply:
Interesting 💡